src/games/strategy/triplea/delegate/GameStepPropertiesHelper.java

```diff
@@ -132,15 +132,15 @@ public static boolean isNonCombatMove(final GameData data, final boolean doNotTh
    * Fire rockets after phase is over. Normally would occur after combat move for WW2v2 and WW2v3, and after noncombat
    * move for WW2v1.
    */
-  static boolean isFireRockets(final GameData data) {
+  static boolean isFireRockets(final GameData data, final boolean moveDelegate) {
```

Comment:
I don't understand why you are asking for this? Why would you want two methods to do basically the same function, only slightly different in different rule sets.